start_admin_sshd.sh

```diff
@@ -47,6 +47,17 @@ get_user_data_keys() {
     ( IFS=$'\n'; echo "${valid_keys[*]}" )
 }
 
+# Export proxy environment variables for all users' login shells
+# Match the values of the proxy environment variables given to the admin container
+install_proxy_profile() {
+  local -r PROXY_ENV_VAR_SETUP_SCRIPT="/etc/profile.d/br-proxy-env.sh"
+  cat <<EOF > ${PROXY_ENV_VAR_SETUP_SCRIPT}
```

Comment:
I think we lose clarity for the sake of brevity here. What do you think about calling this: 

```suggestion
  local -r proxy_profile="/etc/profile.d/bottlerocket-proxy-settings.sh"
  cat > "$proxy_profile" <<EOF
```